### ISSUE TITLE
🐛 Bug: Copy record URL from expanded form is not encoded

### ISSUE THREAD

**Issue by o1lab** [nocodb/nocodb#4392]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

<img width="1151" alt="Screenshot 2022-11-14 at 12 44 05" src="https://user-images.githubusercontent.com/5435402/201663437-69af1cb9-7600-419a-95ea-f6743dac339b.png">

### Desired Behavior

- Record URL should be encoded.

### Project Details

- On latest.

**Comment by bcakmakoglu** [comment]
I can't reproduce this issue on develop.
If I use a table with the name `Producer Foo` (just as an example), I receive `http://localhost:3000/#/nc/p_wiukyreqe052rf/table/Producer%20Foo/Grid%20view?rowId=recQI4iMKMH2yLG0g` as the copied URL.
It is encoded with `%20` where whitespaces would be.

Seems like this was fixed by @wingkwong this morning. (Commit: e0e655270eeb70f0a1e2b14b2e4a30c778a7b594)

```ts
copy(
    encodeURI(
      `${dashboardUrl?.value}#/${route.params.projectType}/${route.params.projectId}/${route.params.type}/${meta.value?.title}${
        props.view ? `/${props.view.title}` : ''
      }?rowId=${primaryKey.value}`,
    ),
  )
```

**Comment by o1lab** [comment]
Ah good. Thank you.

**Comment by wingkwong** [comment]
Yes. I've fixed that in the morning. Closing this issue as it's duplicate as https://github.com/nocodb/nocodb/issues/4328
